src/Components/Directory.Build.targets

```diff
@@ -30,4 +41,56 @@
                               PackagePath="buildTransitive\$(TargetFramework)\$(PackageId).targets" />
     </ItemGroup>
   </Target>
+
+  <PropertyGroup>
+    <ConfigurationSchemaGeneratorPath>$(ArtifactsBinDir)ConfigurationSchemaGenerator\$(Configuration)\$(NetCurrent)\ConfigurationSchemaGenerator.dll</ConfigurationSchemaGeneratorPath>
```

Comment:
This seems a bit fragile, we could call GetTargetPath instead to get the real path by calculating it, but it's probably fine for now.